### ISSUE TITLE
Change package names from org.deri.* to something else

### ISSUE THREAD

**Issue by cygri** [tarql/tarql#52]
DERI no longer exists, and the Tarql contributors are no longer affiliated with its successor organisations.

`com.github.tarql`?

The sooner this is done, the better.

**Comment by akuckartz** [comment]
:+1: for not depending on deri. :-1: for depending g on github.

**Comment by cygri** [comment]
@akuckartz Fair point. The best option would be to register a domain for the project, but I'm spending too much money on domains already :-)

Maybe it's actually not that urgent.
